### PR TITLE
Adding select list elements JS examples + docs

### PR DESCRIPTION
**Pull request by fufu930**
**Thanks for contributing to the Selenium site and documentation!**
**A PR well described will help maintainers to review and merge it quickly**

Before submitting your PR, please check our [contributing](https://selenium.dev/documentation/en/contributing/) guidelines.
Avoid large PRs, and help reviewers by making them as simple and short as possible.

### Types of changes

- [ ] Change to the site (I have double-checked the Netlify deployment, and my changes look good)
- [x] Code example added (and I also added the example to all translated languages)
- [ ] Improved translation
- [ ] Added new translation (and I also added a notice to each document missing translation)

### Checklist

- [ ] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [ ] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.

### CODE REVIEW

File: examples/javascript/test/select/selectListTest.spec.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+            const optionList = await select.getOptions()
+            assert.equal(optionList.length, optionElements.length)
+            for (var index = 0; index < optionList.length; index++) {
+                assert.equal(await optionList[index].getText(), await optionElements[index].getText())
+            }
+
+            const selectedOptionList = await select.getAllSelectedOptions()
+            const expectedSelection = [eggElement, sausageElement]
+            assert.equal(expectedSelection.length, selectedOptionList.length)
+            for (var index = 0; index < selectedOptionList.length; index++) {
```
Comment (harsha509) [inline examples/javascript/test/select/selectListTest.spec.js]: for(const index in selectedOptionList) { ... }

File: examples/javascript/test/select/selectListTest.spec.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+            const hamElement = await driver.findElement(By.css('option[value=ham]'))
+            const gravyElement = await driver.findElement(By.css("option[value='onion gravy']"))
+            const eggElement = await driver.findElement(By.css('option[value=eggs]'))
+            const sausageElement = await driver.findElement(By.css("option[value='sausages']"))
+
+            const optionElements = await selectElement.findElements(By.css('option'))
+            const optionList = await select.getOptions()
+            assert.equal(optionList.length, optionElements.length)
+            for (var index = 0; index < optionList.length; index++) {
```
Comment (harsha509) [inline examples/javascript/test/select/selectListTest.spec.js]: for(const index in optionList) { ... }

File: examples/javascript/test/select/selectListTest.spec.js [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+            await select.deselectByValue('sausages')
+            assert.equal(false, await eggElement.isSelected())
+            assert.equal(false, await sausageElement.isSelected())
+        })
+
+        it('Try selecting disabled option', async function () {
+            const selectElement = await driver.findElement(By.name('single_disabled'))
+            const select = await new Select(selectElement)
+
+            await assert.rejects(async () => { await select.selectByValue("disabled") }, {
```
Comment (harsha509) [inline examples/javascript/test/select/selectListTest.spec.js]: indentation is too much. While rendering in code samples it takes lot more space!

Can you set indent to indent and tab space to 2!

**Review by harsha509 — CHANGES_REQUESTED** [review]
Hi @hanceanic,

Thank you for the PR!

I've provided few comments, could you please look into it! 

Thanks!

File: examples/javascript/test/select/selectListTest.spec.js (hunk 3 above)
Comment (fufu930) [reply examples/javascript/test/select/selectListTest.spec.js]: @harsha509 Alright, committed those

**Review by harsha509 — APPROVED** [review]

**Comment by harsha509** [comment]
Thank you @hanceanic !

**Pull request merged**